client/src/components/SignUp/register.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import axios from 'axios';
 class Register extends React.Component {
   constructor(props){
```

Comment:
I've found it useful to split up API calls into a separate file so that I don't have axios spread out into a bunch of different components. It's just a way to keep things a little more organized.